.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: required 
+language: generic 
+compiler:
+  - gcc
+notifications:
+  email:
+    on_success: change
+    on_failure: always
+    recipients:
+      - ros-contributions@amazon.com
+env:
+  matrix:
+    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu UPSTREAM_WORKSPACE=file $ROSINSTALL_FILENAME="robot_ws/.rosinstall"
+    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu UPSTREAM_WORKSPACE=file $ROSINSTALL_FILENAME="robot_ws/.rosinstall"
+    - ROS_DISTRO="lunar"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu UPSTREAM_WORKSPACE=file $ROSINSTALL_FILENAME="robot_ws/.rosinstall"
```

Comment:
Please remove lunar and melodic. RoboMaker does not yet support these and attempting to build is potentially misleading to customers.